resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/HadoopConfBootstrap.scala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.deploy.k8s
+
+import java.io.File
+
+import scala.collection.JavaConverters._
```

Comment:
Imports are not consistent with the rest of the project. Order should be as follows everywhere:

1. java.io.*
2. Empty Space
3. Everything that isn't java.io.* or org.apache.spark.* (this includes scala.*)
4. Empty space
5. org.apache.spark.*

Please look over all files and fix all imports.

Reply:
I was curious about the import order. According to http://spark.apache.org/contributing.html, the recommended import order is slightly different. scala.* and other 3rd parties libraries are separated by an empty space. Do we know which one is correct?

> In addition, sort imports in the following order (use alphabetical order within each group):
>
> - java.* and javax.*
> - scala.*
> - Third-party libraries (org.*, com.*, etc)
> - Project classes (org.apache.spark.*)

An example from the same page:
```
import java.*
import javax.*
 
import scala.*
 
import *
 
import org.apache.spark.*
```

Reply:
Actually @kimoonkim and I think our code is incorrect in most places.

Reply:
@mccheah Cool. Should we then follow the import order suggested in http://spark.apache.org/contributing.html going forward?

Reply:
Yes we should. We can fix the ordering as we merge upstream.